### PR TITLE
Skip required buildkite check for pre-release pull requests

### PR DESCRIPTION
**Pull request by alxmyth**
The main buildkite job is fairly long running and blocks a quick land.  I'm of the opinion that landing pre-releases should have the least amount of friction as they add no new functionality and do not break existing consumers.

Example:
image

Alternatively, we could let the job complete and then land once the required check is successful.  It's a tad bit more annoying since the `status` payload is linked to a specific commit and does not reference the pull request.

### CODE REVIEW

File: src/auto-approve-and-land-prereleases.js [hunk 1]
```diff
@@ -24,14 +37,17 @@ module.exports = robot => {
     const isMemberOfFusionOrg = await getIsMemberOfOrg(github, 'fusionjs', pr.user.login);
     if(!(isPrerelease && isMemberOfFusionOrg)) return;
 
+    // Wait for any quick-running statuses to complete
+    await delay(3 * 1000);
```
Comment (alxmyth) [inline src/auto-approve-and-land-prereleases.js:41]: Not elegant, but helps mitigate the race to mark build check as `success`.

Comment (KevinGrandon) [reply src/auto-approve-and-land-prereleases.js:41]: Hmm, this has a funny smell to it. What status are we waiting for? Just feels like we should be able to do something with the github hooks to avoid.

**Review by KevinGrandon — APPROVED** [review]
Not really a fan of the wait and would rather use API hooks if possible, but probably fine to land.

File: src/auto-approve-and-land-prereleases.js (hunk 1 above)
Comment (alxmyth) [reply src/auto-approve-and-land-prereleases.js:41]: Agreed.  This isn't great.  An ideal solution would be to ensure (1) all of the checks are super quick and (2) we wait for all to complete before landing.

In practice, some of our checks are slow (and fairly variable in terms of execution time).  Buildkite checks can take on the order of minutes to complete and are (in theory) unnecessary as pre-releases introduce no new code changes.

I'm not convinced we can achieve these goals without quite a bit of re-work, so this was the quick and dirty solution that's "good enough" (TM).  Delay lets the really quick ones run, and implicitly says "whatever isn't done in the timeframe we deem reasonable doesn't matter".

Alternatively, if we don't care much for quick checks, we can remove this and instead track when all statuses have succeeded and then attempt to land.  Pros: less smell; Cons: potentially much slower pre-releases.

TL;DR - `delay` ensures that the quick-running probots have time to add labels, flag malformed titles, etc. without worrying about the long-running jobs (like buildkite's).

Comment (alxmyth) [reply src/auto-approve-and-land-prereleases.js:41]: FWIW, I'd vote to push for faster pre-releases and revisit down the  road if we feel to need to.

**Comment by alxmyth** [comment]
!merge
